packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -139,9 +139,9 @@ export class BoosterGraphQLDispatcher {
       switch (operationData.operation) {
         case 'query':
         case 'mutation':
-          return this.handleQueryOrMutation(queryDocument, resolverContext)
+          return await this.handleQueryOrMutation(queryDocument, resolverContext)
         case 'subscription':
-          return this.handleSubscription(queryDocument, resolverContext)
+          return await this.handleSubscription(queryDocument, resolverContext)
```

Comment:
This change is unrelated to the purpose of this PR, but when I was fixing the tests I realized that we had a potential bug here (probably introduced after the recent changes for global error handling). When there's an error in `handleQueryOrMutation` or `handleSubscription` methods, the catch in line 146 cannot see the exception and it is not properly handled. /cc @gonzalogarciajaubert @alvaroloes @NickSeagull

Reply:
I don't think this change is on main @javiertoledo but only in this PR (from a previous commit). I mean, the removal of the `await` was done in this PR and it's good that you restored it. You can verify it on the "file changes" tab. This line is not there because it's not modified